Add osu! South East Asia Tournament 4 (oSEAT4) wiki article (#4642)
ppy/osu-wiki: wiki/Tournaments/oSEAT/4/en.md

---
tags:
  - oSEAT
  - oSEAT4
---

# osu! South East Asia Tournament 4

![oSEAT4 logo](img/logo.jpg)

> The **osu! South East Asia Tournament 4** (***oSEAT4***) is a double-elimination 1v1 osu!standard tournament hosted by ![][flag_SG] [phox](https://osu.ppy.sh/users/772295). The tournament is open to all players from all ten [ASEAN member states](https://asean.org/asean/asean-member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![flag_ID] Indonesia, ![flag_LA] Laos, ![flag_MY] Malaysia, ![flag_MM] Myanmar, ![flag_PH] The Phillipines, ![flag_SG] Singapore, ![flag_TH] Thailand, and ![flag_VN] Vietnam) regardless of rank. It is the fourth iteration of the tournament series.

Reviewer: ```suggestion
The **osu! South East Asia Tournament 4** (***oSEAT4***) is a double-elimination 1v1 osu! tournament hosted by ![][flag_SG] [phox](https://osu.ppy.sh/users/772295). The tournament is open to all players from all ten [ASEAN member states](https://asean.org/asean/asean-member-states/) (![][flag_BN] Brunei Darussalam, ![][flag_KH] Cambodia, ![flag_ID] Indonesia, ![flag_LA] Laos, ![flag_MY] Malaysia, ![flag_MM] Myanmar, ![flag_PH] The Phillipines, ![flag_SG] Singapore, ![flag_TH] Thailand, and ![flag_VN] Vietnam) regardless of rank. It is the 4th installment of the osu! South East Asia Tournament.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2020-11-07/2020-11-22 |
| Screening phase | 2020-11-23/2020-11-29 |
| Qualifiers | 2020-11-30/2020-12-06 |

> | Group stage (week 1) | 2020-12-07/2020-12-13 |
> | Group stage (week 2) | 2020-12-14/2020-12-20 |

Reviewer: ```suggestion
| Group Stage week 1 | 2020-12-07/2020-12-13 |
| Group Stage week 2 | 2020-12-14/2020-12-20 |
```

| Round of 32 | 2020-12-21/2020-12-27 |
| Round of 16 | 2020-12-28/2021-01-03 |
| Quarterfinals | 2021-01-04/2021-01-10 |
| Semifinals | 2021-01-11/2021-01-17 |

> | Finals (week 1) | 2021-01-18/2021-01-23 |
> | Finals (week 2) | 2021-01-24/2021-01-30 |

Reviewer: ```suggestion
| Finals week 1 | 2021-01-18/2021-01-23 |
| Finals week 2 | 2021-01-24/2021-01-30 |
```

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 12 months of osu!supporter tag, unique profile badge |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 6 months of osu!supporter tag |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 4 months of osu!supporter tag |

> | 4th place | 2 months of osu!supporter tag |
> | 5th place | 1 month of osu!supporter tag |

Reviewer: ```suggestion
| *4th place* | 2 months of osu!supporter tag |
| *5th place* | 1 month of osu!supporter tag |
```

> ![Profile badge](img/Badge.png)

Reviewer: ```suggestion
![](img/Badge.png "oSEAT4 winner badge")
```

## Organization

> oSEAT4 was run by various osu! community members predominantly hailing from South East Asian countries.

Reviewer: ```suggestion
oSEAT4 is run by various osu! community members predominantly hailing from South East Asian countries.
```

> | Position | Members |

Reviewer: ```suggestion
| Position | Member(s) |
```

| :-- | :-- |
| Host | ![][flag_SG] [phox](https://osu.ppy.sh/users/772295) |

> | Map selector | ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), ![][flag_SG] [Fragmented](https://osu.ppy.sh/users/847434), ![][flag_TH] [Seaweed](https://osu.ppy.sh/users/5151214), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498), ![][flag_US] [Not A Function](https://osu.ppy.sh/users/4773094) |

Reviewer: ```suggestion
| Mappool selector | ![][flag_SG] [phox](https://osu.ppy.sh/users/772295), ![][flag_SG] [Fragmented](https://osu.ppy.sh/users/847434), ![][flag_TH] [Seaweed](https://osu.ppy.sh/users/5151214), ![][flag_SG] [Ayucchi](https://osu.ppy.sh/users/7823498), ![][flag_US] [Not A Function](https://osu.ppy.sh/users/4773094) |
```

| Graphic designer | ![][flag_SG] [megumic](https://osu.ppy.sh/users/7537133) |
| Spreadsheet manager | ![][flag_SG] [phox](https://osu.ppy.sh/users/772295) |
| Wiki editor | ![][flag_ID] [Niva](https://osu.ppy.sh/users/197805) |

## Links

- **[Main spreadsheet](https://docs.google.com/spreadsheets/d/16LDQYYDwd7obzSgLijTV1hBD7QOZE_2Ibe9ZLqTWQNw/edit#gid=1373445856)**
- [Forum thread](https://osu.ppy.sh/community/forums/topics/1177376)
- [Discord server](https://discord.gg/sVHndPU)
- [Livestream channel](https://www.twitch.tv/oseatournament)

## Participants

Listed below are players who qualified into the Group stage (along with their respective seeding and group placement) out of 179 registered players in total.

> | Group | Player 1 (Seed A) | Player 2 (Seed B) | Player 3 (Seed C) | Player 4 (Seed D) | Player 5 (Seed E) | Player 6 (Seed F) |

Reviewer: ```suggestion
| Group | Seed A | Seed B | Seed C | Seed D | Seed E | Seed F |
```
should be fine this way

¹ *![][flag_ID] [ngonk](https://osu.ppy.sh/users/12030070) initially qualified for this Group stage berth, but they decided to forfeit from the tournament during the running of the Group stage. Under the discretion of staff members and all other affected players, ![][flag_MY] [Heya](https://osu.ppy.sh/users/11379332) was later admitted to the Group stage in their spot.*

² *![][flag_ID] [cfood](https://osu.ppy.sh/users/8626190) initially qualified for this Group stage berth, but they decided to forfeit from the tournament during the running of the Group stage. Under the discretion of staff members and all other affected players, ![][flag_PH] [Revillica](https://osu.ppy.sh/users/9806095) was later admitted to the Group stage in their spot.*

³ *Despite having the ![][flag_US] United States flag displayed on their profile, [\_Kolin](https://osu.ppy.sh/users/7249644) is eligible to participate in the tournament as they are in possession of a valid ![][flag_PH] [Phillipine passport](https://en.wikipedia.org/wiki/Philippine_passport).*

## Mappools

> ### Group stage (week 2)

Reviewer: ```suggestion
### Group Stage week 2
```

  - [VY1 - Cyber Thunder Cider (tutuhaha) \[Cider\]](https://osu.ppy.sh/beatmapsets/47146#osu/146610)
  - [Function Phantom - Variable (Chaoslitz) \[Deluxe\]](https://osu.ppy.sh/beatmapsets/751774#osu/1582594)
  - [Kabocha - EmbryO (Kawashiro) \[Garden's Expert\]](https://osu.ppy.sh/beatmapsets/947018#osu/1990062)
- Hidden
  - [Natsuiro Matsuri - Marigold (Ayucchi) \[Bittersweet\]](https://osu.ppy.sh/beatmapsets/1167380#osu/2435698)
  - [kors k - SigSig (Reiji-RJ) \[Another\]](https://osu.ppy.sh/beatmapsets/27677#osu/92684)
- HardRock
  - [Grand Thaw - SOLROS (Lunala) \[Fantasia\]](https://osu.ppy.sh/beatmapsets/1136592#osu/2376019)
  - [siromaru - Absurd Gaff (spro) \[Another\]](https://osu.ppy.sh/beatmapsets/17563#osu/66662)
- DoubleTime
  - [Shirakami Fubuki - Say! Fanfare! (Bokamin) \[Insane\]](https://osu.ppy.sh/beatmapsets/1277448#osu/2658704)
  - [Giga-P - Electric Angel (koreapenguin) \[Sonnyc's Insane\]](https://osu.ppy.sh/beatmapsets/172900#osu/417898)
- FreeMod
  - [M2U - A Bella! (feat. Lucy) (Okoratu) \[Battle's Extra\]](https://osu.ppy.sh/beatmapsets/1158767#osu/2418106)
  - [M2U - Memory of Beach (milr\_) \[Mx\]](https://osu.ppy.sh/beatmapsets/1120861#osu/2342057)

> ### Group stage (week 1)

Reviewer: ```suggestion
### Group Stage week 1
```

### Qualifiers

**[Download the mappack here! (52 MB)](https://drive.google.com/uc?export=download&id=1MjH9DYH4XODeh1Z0jHrZhYJeKd6hozn4)**

- NoMod
  - [MY FIRST STORY - Mukoku (AirinCat) \[Extra\]](https://osu.ppy.sh/beatmapsets/1047546#osu/21896537)
  - [Tyrfing - Verflucht (Muya) \[Another\]](https://osu.ppy.sh/beatmapsets/965125#osu/2045906)
  - [Zutto Mayonaka de Ii no ni. - Byoushin o Kamu (Irohas) \[moph's Extra\]](https://osu.ppy.sh/beatmapsets/833772#osu/1746520)
- Hidden
  - [USAO - Knight Rider (RLC) \[Extra\]](https://osu.ppy.sh/beatmapsets/1156969#osu/2414318)
  - [Chata - len (Cami) \[Nightmare Descends\]](https://osu.ppy.sh/beatmapsets/857816#osu/1792255)
- HardRock
  - [Linkin Park - Session (h4d0uk3n1) \[Katherine\]](https://osu.ppy.sh/beatmapsets/669536#osu/1488047)
  - [Dance Gavin Dance - Tree Village (Dilectus) \[Extra\]](https://osu.ppy.sh/beatmapsets/703666#osu/1488698)
- DoubleTime
  - [Buono! - Winter Story (Nakagawa-Kanon) \[Insane\]](https://osu.ppy.sh/beatmapsets/40425#osu/128303)
  - [dj TAKA - Colors -sasakure.UK Futurelogic Remix- (\_wa) \[Hyper\]](https://osu.ppy.sh/beatmapsets/317439#osu/1319822)

## Match results

> ### Group stage (week 1)

Reviewer: ```suggestion
### Group Stage week 1
```
